File: android/app/src/main/java/com/rafaeltmbr/stopwatch/domain/entities/StopwatchState.kt
package com.rafaeltmbr.stopwatch.domain.entities

data class StopwatchState(
    val status: Status = Status.INITIAL,
    val milliseconds: Long = 0L,
    val completedLaps: List<Lap> = emptyList(),
    val completedLapsMilliseconds: Long = 0L,
)

File: ios/Stopwatch/core/entities/StopwatchState.swift
struct StopwatchState {
    let status: Status
    let milliseconds: Int
    let completedLaps: [Lap]
    let completedLapsMilliseconds: Int
    
    init(
        status: Status = .initial,
        milliseconds: Int = 0,
        completedLaps: [Lap] = [],
        completedLapsMilliseconds: Int = 0
    ) {
        self.status = status
        self.milliseconds = milliseconds
        self.completedLaps = completedLaps
        self.completedLapsMilliseconds = completedLapsMilliseconds
    }
}
